packages/stream_chat_persistence/README.md

```diff
@@ -0,0 +1,14 @@
+# stream_chat_persistence
+
+A new Flutter package.
```

Comment:
I think we should update this with a short description of the package and its intended use. 

@imtoori and I can help with this.